bool.go

```diff
@@ -0,0 +1,45 @@
+package nulltypes
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+)
+
+//NullBool is a wrapper around bool
+type NullBool struct {
+	Bool  bool
+	Valid bool
+}
+
+// Bool method to get NullBool object from bool
+func Bool(Bool bool) NullBool {
+	return NullBool{Bool, true}
+}
+
+//MarshalJSON method is called by json.Marshal,
```

Comment:
Comments not properly formatted 

```suggestion
// MarshalJSON method is called by json.Marshal,
```